_data/simple-icons.json

```diff
@@ -5951,8 +5951,8 @@
         },
         {
             "title": "Home Assistant",
-            "hex": "41BDF5",
-            "source": "https://github.com/home-assistant/assets/blob/1e19f0dca208f0876b274c68345fcf989de7377a/logo/logo-small.png",
+            "hex": "18BCF2",
+            "source": "https://github.com/home-assistant/assets/blob/0f69676da79c3881e7dfca6d6c0a5739f88273d8/logo/home-assistant-logo.zip",
             "license": {
                 "type": "CC-BY-NC-SA-4.0"
             }
```

Comment:
As mentioned in https://github.com/home-assistant/assets/tree/0f69676da79c3881e7dfca6d6c0a5739f88273d8/logo#copyright-notice, we should remove the license.

> Please note that this logo is not released under the CC license. All rights reserved.